metrics-facade-prometheus/src/test/java/com/ringcentral/platform/metrics/reporters/prometheus/PrometheusMetricsExporterTest.java

```diff
@@ -199,13 +219,34 @@ public void export() {
             null,
             null,
             null,
+            null,
             List.of(DIMENSION_1, DIMENSION_2, DIMENSION_3, "quantile"),
             List.of("dimension_1_value", "dimension_2_value", "dimension_3_value", "0.75"),
             14.0));
 
         instanceSamples.add(instanceSample);
+        CollectorRegistry collectorRegistry = new CollectorRegistry(true);
+
+        SimpleCollectorRegistryPrometheusInstanceSamplesProvider collectorRegistryInstanceSamplesProvider = new SimpleCollectorRegistryPrometheusInstanceSamplesProvider(
+            name("defaultRegistry"), // optional prefix
+            new SampleNameFilter.Builder().nameMustNotStartWith("counter_1").build(), // optional filter
```

Comment:
Let's remove both filters - it's not clear if it's part of the test or not, they don't seem to filter anything.

Reply:
Actually. the "created" filters removes the unpredictable time based sample.
However, it seems that adding that additional instance samples provider is not needed because SimpleCollectorRegistryPrometheusInstanceSamplesProvider is already tested in SimpleCollectorRegistryPrometheusInstanceSamplesProviderTest. 
Let's just remove that provider.

Reply:
Removed the provider usage from the test.